Changes "default-mobile" to mobian in branding
--- a/branding/default-mobile/branding.desc
+++ b/branding/default-mobile/branding.desc
@@ -92,8 +92,8 @@
 # are visible as buttons there if the corresponding *show* keys
 # are set to "true" (they can also be overridden).
 strings:
-    productName:         "NextGenMobileLinuxDistro"
-    shortProductName:    NextGenMobileLinuxDistro
+    productName:         "Mobian"
+    shortProductName:    Mobian
 
 # These images are loaded from the branding module directory.
 #
