Modifications required to install Mobian using calamares-mobile
--- a/modules/mobile/PartitionJob.cpp
+++ b/modules/mobile/PartitionJob.cpp
@@ -77,9 +77,11 @@
     QString cryptDev = "/dev/mapper/" + cryptName;
     QString passwordStdin = m_password + "\n";
     QString dev = m_targetDeviceRoot;
+    QString devNoPart = dev.chopped(2);
 
     QList< QPair< QStringList, QString > > commands = {
         { { "mkdir", "-p", pathMount }, QString() },
+        { { "parted", devNoPart, "mkpart", "primary", "ext4", "268M", "4.9G", "-s"}, QString()},
     };
 
     if ( m_isFdeEnabled )
--- a/modules/mobile/wait.qml
+++ b/modules/mobile/wait.qml
@@ -38,7 +38,7 @@
             anchors.topMargin: 150
             wrapMode: Text.WordWrap
             text: "Formatting and mounting target partition. This may" +
-                  " take up to 20 seconds, please be patient."
+                  " take up to two minutes, please be patient."
             width: 500
         }
     }
