Modifications required to install Mobian using calamares-mobile
Index: calamares-settings-mobian/modules/mobile/PartitionJob.cpp
===================================================================
--- calamares-settings-mobian.orig/modules/mobile/PartitionJob.cpp	2020-12-17 08:46:13.240000000 +0000
+++ calamares-settings-mobian/modules/mobile/PartitionJob.cpp	2020-12-17 08:46:13.239000000 +0000
@@ -77,9 +77,11 @@
     QString cryptDev = "/dev/mapper/" + cryptName;
     QString passwordStdin = m_password + "\n";
     QString dev = m_targetDeviceRoot;
+    QString devNoPart = dev.chopped(2);
 
     QList< QPair< QStringList, QString > > commands = {
         { { "mkdir", "-p", pathMount }, QString() },
+	{ { "parted", devNoPart, "mkpart", "primary", "ext4", "268M", "4.9G", "-s"}, QString()},
     };
 
     if ( m_isFdeEnabled )
Index: calamares-settings-mobian/modules/mobile/ssh_confirm.qml
===================================================================
--- calamares-settings-mobian.orig/modules/mobile/ssh_confirm.qml	2020-12-17 08:46:13.240000000 +0000
+++ calamares-settings-mobian/modules/mobile/ssh_confirm.qml	2020-12-17 08:46:13.239000000 +0000
@@ -29,7 +29,7 @@
               "<br>" +
               "With 'enable', you will be asked for a second username and" +
               " password. You will be able to login to the SSH server with" +
-              " these credentials via USB (172.16.42.1), Wi-Fi and possibly" +
+              " Wi-Fi and possibly" +
               " cellular network. It is recommended to replace the password" +
               " with an SSH key after the installation.<br>" +
               "<br>" +
Index: calamares-settings-mobian/modules/mobile/wait.qml
===================================================================
--- calamares-settings-mobian.orig/modules/mobile/wait.qml	2020-12-17 08:46:13.240000000 +0000
+++ calamares-settings-mobian/modules/mobile/wait.qml	2020-12-17 08:46:13.239000000 +0000
@@ -38,7 +38,7 @@
             anchors.topMargin: 150
             wrapMode: Text.WordWrap
             text: "Formatting and mounting target partition. This may" +
-                  " take up to 20 seconds, please be patient."
+                  " take up to two minutes, please be patient."
             width: 500
         }
     }
